source/transactions-convenient-api/tests/README.md

```diff
@@ -41,8 +41,73 @@ If possible, drivers should implement these tests without requiring the test run
 the retry timeout. This might be done by internally modifying the timeout value used by `withTransaction` with some
 private API or using a mock timer.
 
+### Retry Backoff is Enforced
+
+Drivers should test that retries within `withTransaction` do not occur immediately.
+
+1. let `client` be a `MongoClient`
+2. let `coll` be a collection
+3. Now, run transactions without backoff:
+    1. Configure the random number generator used for jitter to always return `0` -- this effectively disables backoff.
+
+    2. Configure a fail point that forces 13 retries like so:
+
+        ```python
+           set_fail_point(
+              {
+                  "configureFailPoint": "failCommand",
+                  "mode": {
+                      "times": 13
+                  },  # sufficiently high enough such that the time effect of backoff is noticeable
+                  "data": {
+                      "failCommands": ["commitTransaction"],
+                      "errorCode": 251,
+                  },
+              }
+           )
+        ```
+
+        > [!NOTE]
+        > errorCode 251 is NoSuchTransaction.
```

Comment:
This isn't formatting for me?  Maybe it's something to do with the indentation?

Reply:
you were right about it being indentation! but unindenting it messes with the formatting of the bullets below it so no fancy alerts this time ;-; 
i gotta keep this fancy alerts thing in mind for a future opportunity -- I want to use it now haha
(made it a normal comment in [1f04505](https://github.com/mongodb/specifications/pull/1851/commits/1f04505c85cb72dfe921f1ee70033fc9f07ed336))